### PR TITLE
searchFields/database migrations

### PR DESCRIPTION
**Pull request by SRattacasa**
- New migration added to include additional groomer filtering options

### CODE REVIEW

**Comment by SRattacasa** [comment]
- Changing directory to the /config path, I created a new database migration to add a "dogs" and "cats" boolean field to the groomers database table
- This is to be used for filtering out groomers in the search component based on the animals serviced

**Comment by SRattacasa** [comment]
- Had to do some troubleshooting of the npm lint script to include the --fix flag 

The automated testing was failing on some eslint errors, so I added the --fix flag and it seems to have resolved them. It's passing the tests now.

File: data/migrations/20210106195157_add_animals_to_groomers.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+exports.up = (knex) => {
+    return knex.schema.table('groomers', function (table) {
+        table.boolean('dogs')
+        table.boolean('cats')
+    });
```
Comment (coryortega) [inline data/migrations/20210106195157_add_animals_to_groomers.js:5]: check what it defaults to maybe '.default()'

**Comment by SRattacasa** [comment]
- Ran against development database
- Verified the column created successfully
- Validated the seed file ran and created the desired values

File: data/migrations/20210106195157_add_animals_to_groomers.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  };
+
+
+exports.down = (knex) => {
+    return knex.schema.table('groomers', function (table) {
+      table.dropColumn('dogs').defaultTo(false);
+      table.dropColumn('cats').defaultTo(false);
```
Comment (coryortega) [inline data/migrations/20210106195157_add_animals_to_groomers.js:12]: .defaultTo should be on the exports.up

File: data/migrations/20210106195157_add_animals_to_groomers.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+exports.up = (knex) => {
+    return knex.schema.table('groomers', function (table) {
+        table.boolean('dogs')
+        table.boolean('cats')
```
Comment (coryortega) [inline data/migrations/20210106195157_add_animals_to_groomers.js:4]: add the .defaultTo up here

**Review by coryortega — CHANGES_REQUESTED** [review]
Looks good! Just change the defaultTo to be on the exports.up function

File: data/migrations/20210106195157_add_animals_to_groomers.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+        table.boolean('dogs').defaultTo(false);
+        table.boolean('cats').defaultTo(false);
+    });
+  };
+
+
+exports.down = (knex) => {
+    return knex.schema.table('groomers', function (table) {
+      table.dropColumn('dogs').defaultTo(false);
+      table.dropColumn('cats').defaultTo(false);
```
Comment (coryortega) [inline data/migrations/20210106195157_add_animals_to_groomers.js:12]: Get rid of the .defaultTo on the down

**Review by coryortega — CHANGES_REQUESTED** [review]

**Review by coryortega — APPROVED** [review]

**Review by AustinKelsay — APPROVED** [review]

**Pull request merged**